### DISCUSSION TITLE
Winget...

### DISCUSSION THREAD

**Discussion by KnifMelti** [Romanitho/Winget-Install#41] [Q&A] [answered]
...sometimes it just sits there (any chance a timeout could be implemented? - this finally installed after 5 minutes):

![image](https://user-images.githubusercontent.com/102996177/168324989-e55e8308-36b0-4b52-aed7-89f40f6abea7.png)

EDIT: But, we have the Max allowed install time setting in SCCM - not necessary then...

**Comment by Romanitho** [comment, marked as answer]
Its depending on the app of course. But maybe in the improvements list we could have a better info popup. But quite hard to implement cause it would be based on the winget cmd stdout, see what i meen ?
